src/modules/webhook/jitsi/jitsi-webhook.service.spec.ts

```diff
@@ -0,0 +1,366 @@
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { Test, TestingModule } from '@nestjs/testing';
+import { JitsiWebhookService } from './jitsi-webhook.service';
+import { JitsiParticipantLeftWebHookPayload } from './interfaces/JitsiParticipantLeftData';
+import { ConfigService } from '@nestjs/config';
+import { RoomUseCase } from '../../room/room.usecase';
+import { RoomUserUseCase } from '../../room/room-user.usecase';
+import { Logger } from '@nestjs/common';
+import * as crypto from 'crypto';
+import { JitsiGenericWebHookEvent } from './interfaces/JitsiGenericWebHookPayload';
+import { Room } from '../../room/room.domain';
+
+jest.mock('crypto', () => {
+  const originalModule = jest.requireActual<typeof import('crypto')>('crypto');
+  return {
+    ...originalModule,
+    createHmac: jest.fn().mockImplementation(() => ({
+      update: jest.fn().mockReturnThis(),
+      digest: jest.fn().mockReturnValue('mocked-signature'),
+    })),
+    timingSafeEqual: jest.fn() as jest.MockedFunction<
+      typeof crypto.timingSafeEqual
+    >,
+  };
+});
+
+describe('JitsiWebhookService', () => {
+  let service: JitsiWebhookService;
+  let roomUseCase: DeepMocked<RoomUseCase>;
+  let roomUserUseCase: DeepMocked<RoomUserUseCase>;
+  let configService: DeepMocked<ConfigService>;
+
+  const minimalRoom = new Room({
+    id: 'test-room-id',
+    maxUsersAllowed: 10,
+    hostId: 'host-id',
+  });
+
+  beforeEach(async () => {
+    roomUseCase = createMock<RoomUseCase>();
+    roomUserUseCase = createMock<RoomUserUseCase>();
+    configService = createMock<ConfigService>();
+
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [
+        JitsiWebhookService,
+        {
+          provide: ConfigService,
+          useValue: configService,
+        },
+        {
+          provide: RoomUseCase,
+          useValue: roomUseCase,
+        },
+        {
+          provide: RoomUserUseCase,
+          useValue: roomUserUseCase,
+        },
+      ],
+    }).compile();
+
+    service = module.get<JitsiWebhookService>(JitsiWebhookService);
+
+    // Mock logger
+    jest.spyOn(Logger.prototype, 'log').mockImplementation(() => undefined);
```

Comment:
Instead of mocking the logger globally, perhaps you could spy on it each time you print a log, so you can verify that the printed message is what we want.